docs/_includes/header.html

```diff
@@ -2,9 +2,8 @@
   <nav class="js-navbar-scroll navbar navbar-expand navbar-dark flex-column flex-md-row td-navbar">
     <a class="navbar-brand" href="{{ site.baseurl }}/{{linkUrlSelected}}">
       <div class="navbar-logo">
-        {% include meshery-logo-text.svg %}<span class="font-weight-bold" style="vertical-align: bottom"
-          >{{ site.title }}</span
-        >
```

Comment:
@msaini0r will you either revert these formatting changes and push that commit or close this PR and try again?